lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -260,6 +271,8 @@ export class PublicClientApplication implements IPublicClientApplication {
      * @returns {Promise.<AuthenticationResult>} - a promise that is fulfilled when this function has completed, or rejected if an error was raised. Returns the {@link AuthResponse} object
      */
     async acquireTokenPopup(request: PopupRequest): Promise<AuthenticationResult> {
+        this.preflightBrowserEnvironmentCheck();
```

Comment:
This is done outside the `try` because we don't want the `this.browserStorage.cleanRequest()` call to run in non-browser environments.

Reply:
@pkanher617 Is it okay if this check (which checks for browser environment and if we're in an iframe) is done outside catch, specifically for the iframe request? My understanding is the `cleanRequest` call is a consideration for the other errors that are potentially thown in the `preflightInteractiveRequest`, but is not needed for the iframe check.